requests_ntlm/requests_ntlm.py

```diff
@@ -55,11 +55,19 @@ def retry_using_http_NTLM_auth(self, auth_header_field, auth_header,
         request = response.request.copy()
 
         # initial auth header with username. will result in challenge
-        msg = "%s\\%s" % (self.domain, self.username) if self.domain else self.username
+        if not self.domain:
+            msg = self.username
+            type1_flags = ntlm.NTLM_TYPE1_FLAGS & \
+                ~ntlm.NTLM_NegotiateOemDomainSupplied
```

Comment:
Let's favour implicit line continuation inside parentheses instead of explicit line continuation:

```python
type1_flags = (
    ntlm.NTLM_TYPE1_FLAGS & ~ntlm.NTLM_NegotiateOemDomainSupplied
)
```